enhancements/local-storage/health-monitoring.md

```diff
@@ -0,0 +1,137 @@
+---
+title: local-device-health-monitoring
+authors:
+  - "@rohantmp"
+reviewers:
+  - "@gnufied"
+  - "@jsafrane”
+  - "@sp98"
+  
+approvers:
+  - "@gnufied"
+  - "@jsafrane"
+creation-date: 2021-07-16
+last-updated: 2021-07-16
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+
+
+# Local Device Health Monitoring
+
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes that we export health metrics for each local device on each node.
+
+## Motivation
+
+Local device vendors expose health information about storage devices via APIs like [SMART](https://en.wikipedia.org/wiki/S.M.A.R.T.). It very useful to know when a storage is about to fail, to differentiate between storage hardware related errors vs purely software ones, etc.
+
+### Goals
+
+- Expose health information about local storage devices in a way that can generate alerts
+
+### Non-Goals
+
+- Align with sig-storage [health monitoring for persistent volumes](https://kubernetes.io/blog/2021/04/16/volume-health-monitoring-alpha-update/)
+  - justification: it is also useful to monitor system drives and CSI only deals with persitent volumes, and migrating to CSI is an extremely long term effort.
+
+## Proposal
```

Comment:
I'd appreciate definition of "device heath" here (or below). It can be very different thing to different people.